### ISSUE TITLE
[Bug]: Keyboard focus is not visible on 'Splitter for packages list' control when High Contrast modes are enabled.

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#11117]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

VS any

### Worked before?

no

### Impact

It bothers me. A fix would be nice

### Repro Steps & Context

See https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1359953

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier
OS & Environment Details:
Microsoft Windows 10 Enterprise Version 21H1 (OS build: 19043.1052)
Microsoft Visual Studio Enterprise 2022 Preview (64-bit)
Version 17.0.0 Preview 1.0 [31404.170.main]
Repro Steps:​
Turn on High Contrast Black/White
Launch latest Visual studio 2022 preview 1
Tab till 'Create new prokect' and press enter
Search for "gRPC Service" project and select it 
Tab till 'Next' button and press enter
Provide proper details and press enter on 'Create' button
Tab till "Solution" and open context menu from keyboard
Tab till 'Manage Nuget Packages for solution' and press enter
Tab till 'Update' tab item and press enter
Search for "gRPC package" and select it.
Tab till 'Splitter for packages list' control and verify keyboard focus is visible or not.
Actual Result:​
Users are unable to see keyboard focus on 'Splitter for packages list' control when High Contrast modes are enabled.
Expected Result:​
Keyboard focus must be visible on 'Splitter for packages list' control even on enabling High Contrast Black/white modes.
User Impact: ​
Low vision users may not able to track keyboard focus if it is not visible on 'Splitter for packages list' control in High Contrast modes.
MAS Reference:​
MAS 4.3.1 No Disruption of Accessibility Features
